app/src/main/java/de/westnordost/streetcomplete/quests/barrier_opening/AddBarrierOpening.kt

```diff
@@ -19,7 +19,10 @@ class AddBarrierOpening(
 
     private val nodeFilter by lazy { """
         nodes with
-            barrier ~ gate|entrance|sliding_gate|swing_gate|wicket_gate|bollard|block
+            (
+                barrier ~ gate|entrance|sliding_gate|swing_gate|wicket_gate|bollard|block
+                or cycle_barrier ~ single|diagonal
```

Comment:
```suggestion
                or barrier = cycle_barrier and cycle_barrier ~ single|diagonal
```

Reply:
sure, if you prefer, although `cycle_barrier=*` won't be found on things that aren't cycle-barrierish...